<!-- -->
<launch>

	<arg name="ars_path_planner_core_node_name" 	default="ars_path_planner_core_node" />
	<arg name="ars_path_planner_manager_node_name" 	default="ars_path_planner_manager_node" />
	<arg name="screen" 	default="screen" />


  <node name="$(arg ars_path_planner_core_node_name)" pkg="ars_path_planner" type="ars_path_planner_core_ros_node.py" output="$(arg screen)" >

  <node name="$(arg ars_path_planner_manager_node_name)" pkg="ars_path_planner" type="ars_path_planner_manager_ros_node.py" output="$(arg screen)" >


  </node>


</launch>
